src/Header/Header.tsx

```diff
@@ -15,29 +15,29 @@ const Header: React.FC<HeaderProps> = props => {
       <div className='top-header'>
       <div className='logo-box'>
         <span className='company-logo'>
-          <img src={globe} alt='Blue geometric globe' className='globe-image' />
-          <h1 className='company-name'>Goin SomewhAIR?</h1>
+          <img src={america} alt='Blue map of USA' className='map-icon' />
+          <h1 className='company-name'>Breathe Am</h1><h1 className='air-text'>air</h1><h1 className='last-letters'>ica</h1>
```

Comment:
Love the new logo and the way you styled the title of our site! It looks very professional!